views/layouts/main.handlebars

```diff
@@ -1,50 +1,45 @@
 <!DOCTYPE html>
```

Comment:
Should we make a favicon version of the botomir icon and add it to this?
We should also get the updated file from Kiara

Reply:
Yes, but she seems really busy right now + her surgery, so im going to wait a bit longer before poking her again. But we will definitely be doing that + updating any other images once it is available.